### ISSUE TITLE
example web3studio.eth.soy not working

### ISSUE THREAD

**Issue by dportabella** [ConsenSys/web3studio-soy#64]
opening the website in Chrome: https://web3studio.eth.soy/
informs of the expired https, 
and the it appears a link named web3studio, with content:
```
ipfs resolve -r /ipfs/QmSC89fFVFFSVC46YmkASGhiwWKhe9MjbmXxceMegeL5zA/ipfs/QmSC89fFVFFSVC46YmkASGhiwWKhe9MjbmXxceMegeL5zA/web3studio: no link named "ipfs" under QmSC89fFVFFSVC46YmkASGhiwWKhe9MjbmXxceMegeL5zA
```
I guess this is not the indented content of https://web3studio.eth.soy/

is this project up-to-date?
can it be used to deploy a website?

**Comment by barlock** [comment]
I don’t believe the project to be maintained anymore. It still functions, a better link is https://web3studio.eth.soy/web3studio/

But as you noted the cert expired. @breakpointer, perhaps could fix that? I’m fairly certain though that if you wanted to maintain the project we could make that happen.

When the project started there were a few other impressions that I don’t remember that used swarm instead of ipfs, I’d be curious if they support it now.
